mk/config.mk

```diff
@@ -411,7 +411,7 @@ CFG_EXTERNAL_DTB_OVERLAY ?= n
 
 # All embedded tests are supposed to be disabled by default, this flag
 # is used to control the default value of all other embedded tests
-CFG_ENABLE_EMBEDDED_TESTS ?= n
+CFG_ENABLE_EMBEDDED_TESTS ?= y
```

Comment:
Why do we need to enable this CFG_ENABLE_EMBEDDED_TESTS? If it's necessary, don't change the default value here. You can add it to platform related conf.mk.